custom/icds_reports/utils/aggregation_helpers/distributed/agg_ls_data.py

```diff
@@ -4,10 +4,10 @@
 from corehq.apps.userreports.util import get_table_name
 from custom.icds_reports.const import AGG_LS_VHND_TABLE, AGG_LS_AWC_VISIT_TABLE, AGG_LS_BENEFICIARY_TABLE
 from custom.icds_reports.utils.aggregation_helpers import transform_day_to_month, month_formatter
-from custom.icds_reports.utils.aggregation_helpers.monolith.base import BaseICDSAggregationHelper
+from custom.icds_reports.utils.aggregation_helpers.distributed.base import BaseICDSAggregationDistributedHelper
 
 
-class AggLsHelper(BaseICDSAggregationHelper):
+class AggLsHelper(BaseICDSAggregationDistributedHelper):
```

Comment:
why was this not distributed already?

Reply:
The same queries were run on both monolith and distributed implementations. The change/move I made was to make it easier to `rm -rf .../monolith/` later

See: https://github.com/dimagi/commcare-hq/pull/25604/files#diff-7fb91312bce0449a4bd7cc9e51062aedL179
https://github.com/dimagi/commcare-hq/blob/a3387e4cb89c41fdc7305e92a8fc97e9a8bc8228/custom/icds_reports/utils/aggregation_helpers/helpers.py#L87-L92